packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1100,16 +1090,18 @@ task(TASK_RUN)
     'deployAll',
     'Deploy all ChugSplash configs before executing your script.'
   )
+  .addFlag('confirm', 'Automatically confirm contract upgrades')
```

Comment:
> Imo, this flag should only be necessary if the user is on a live network. I'd personally find it annoying to include a `--confirm` flag whenever I'm using this command to test an upgrade on the Hardhat network. A solution to this would be:
> 
> 1. Change this flag description to be something like: `Automatically confirm contract upgrades. Only applicable if upgrading on a live network.`
> 2. Inside this task, set `confirm` to `true` if we're on the Hardhat network.
> 
> In the future, we could use the CI deployment task instead of this solution, but I think this is sufficient for now.

Same for this task

Reply:
Looks like you might've forgotten to do #1

Reply:
Huh, seems like the fix for #1 might have been reversed when I rebased didn't realize that could happen.